src/ui/EditorNS/editor.cpp

```diff
@@ -141,37 +139,33 @@ namespace EditorNS
         if (msg.startsWith("[ASYNC_REPLY]")) {
             QRegExp rgx("\\[ID=(\\d+)\\]$");
 
-            if(rgx.indexIn(msg) == -1)
-                return;
-
-            if (rgx.captureCount() != 1)
+            if (rgx.indexIn(msg) == -1 || rgx.captureCount() != 1)
```

Comment:
Messing with on_proxyMessageReceived is going to create a lot of merge conflicts in the webengine2 and webengine2-promise branches I'm working on... It's getting hard to fix conflicts at every PR xD.

I think it's not worth the work, especially because this code is going to disappear anyway. Could you revert these changes, or at least do them on those branches?